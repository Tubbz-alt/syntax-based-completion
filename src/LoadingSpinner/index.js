import LoadingSpinner from "./loading_spinner";
export default LoadingSpinner;
