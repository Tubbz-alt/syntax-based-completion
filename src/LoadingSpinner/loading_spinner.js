import React from "react";
import "./loading_spinner.css";

export default class LoadingSpinner extends React.PureComponent {
  render() {
    return (
      <div className="lds-ellipsis">
        <div />
        <div />
        <div />
        <div />
      </div>
    );
  }
}
