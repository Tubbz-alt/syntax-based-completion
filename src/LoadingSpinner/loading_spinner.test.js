import React from "react";
import { shallow } from "enzyme";

import LoadingSpinner from "./";

describe("<LoadingSpinner/>", () => {
  it("shallow render", () => {
    const wrapper = shallow(<LoadingSpinner />);
    expect(wrapper.hasClass("lds-ellipsis"));
    expect(wrapper).toMatchSnapshot();
  });
});
