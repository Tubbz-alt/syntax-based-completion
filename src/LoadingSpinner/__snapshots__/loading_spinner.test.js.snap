// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadingSpinner/> render 1`] = `"<div class=\\"CompletionInput\\"><input class=\\" CompletionInput__input\\" delay=\\"100\\" value=\\"\\" readonly=\\"\\" placeholder=\\"Parsing syntax...\\"/></div>"`;

exports[`<LoadingSpinner/> shallow 1`] = `ShallowWrapper {}`;

exports[`<LoadingSpinner/> shallow render 1`] = `ShallowWrapper {}`;
